src/components/Filters.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import cn from 'classnames';
+import PropTypes from 'prop-types';
+
+const Filters = ({ handleTypeOfFilter, typeOfFilter }) => (
+  <ul className="filters">
+    <li>
+      <a
+        href="#/"
+        className={cn({
+          selected: typeOfFilter === 'all',
```

Comment:
вынеси строки в константы
```suggestion
          selected: typeOfFilter === FILTER_TYPES.ALL,
```